packages/react-components/src/AddressSmall.tsx

```diff
@@ -6,11 +6,12 @@ import { Address, AccountId } from '@polkadot/types/interfaces';
 
 import React from 'react';
 import styled from 'styled-components';
+import { Popup } from '@polkadot/react-components';
 
 import AccountIndex from './AccountIndex';
 import AccountName from './AccountName';
 import IdentityIcon from './IdentityIcon';
-
+import { Icon } from '@polkadot/react-components';
```

Comment:
may be import together
import { Icon, Popup} from '@polkadot/react-components';